src/couch/src/couch_util.erl

```diff
@@ -46,15 +46,16 @@
 -define(FLUSH_MAX_MEM, 10000000).
 
 -define(BLACKLIST_CONFIG_SECTIONS, [
-    <<"daemons">>,
-    <<"external">>,
-    <<"httpd_design_handlers">>,
-    <<"httpd_db_handlers">>,
-    <<"httpd_global_handlers">>,
-    <<"native_query_servers">>,
-    <<"os_daemons">>,
-    <<"query_servers">>,
-    <<"feature_flags">>
+    <<"^daemons$">>,
+    <<"^external$">>,
+    <<"^httpd_design_handlers$">>,
+    <<"^httpd_db_handlers$">>,
+    <<"^httpd_global_handlers$">>,
+    <<"^native_query_servers$">>,
+    <<"^os_daemons$">>,
+    <<"^query_servers$">>,
+    <<"^feature_flags$">>,
+    <<"^tracing\..*$">>
```

Comment:
Adding the `\.` to this regexp means that we're allowing HTTP access to the main `tracing` section which seems like something even more important to protect than the samplers and action bits (granted action bits are dynamically compiled so also important for those).

Reply:
Good point about adding `tracing` into `BLACKLIST_CONFIG_SECTIONS` although I don't quite understand a problem with regex itself. The current regexp should block sections named `<tracing> + <dot> + <anything else>`.
```
2> re:run(<<"tracing.all-docs">>, <<"^tracing\..*$">>).
{match,[{0,16}]}
```

Reply:
Sure. I was just saying removing the `\.` would also match the `[tracing]` main section as well. Having it as two matches is fine as well.

Reply:
I wanted to allow modification of `tracing.samplers` since it would make it possible to enable/disable pre-configured tracing rules via http. I am going to add a second regex to block `<<"^tracing$">>`. Let me know if you prefer to block `tracing.samplers`.